examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -246,13 +212,15 @@ config("silabs-wifi-config") {
 }
 
 source_set("efr32-common") {
-  deps = []
+  deps = [ "${chip_root}/src/platform/silabs/provision:provision-headers" ]
   defines = []
   public_deps = []
   public_configs = [
     ":efr32-common-config",
     "${efr32_sdk_build_root}:silabs_config",
     ":chip_examples_project_config",
+
+    "${chip_root}/src/platform/silabs/provision:provision-config",
```

Comment:
we don't need this on the 917 side?
I don't think we needs this since it is a public config of the headers.

Reply:
maybe, ill try it